app/src/main/java/com/tpp/theperiodpurse/ui/calendar/CalendarScreen.kt

```diff
@@ -227,7 +227,7 @@ fun CalendarScreenLayout(
                     modifier = Modifier
                         .padding(horizontal = 12.dp)
                         .semantics { contentDescription = "Calendar" },
-                    contentPadding = PaddingValues(bottom = 48.dp),
+                    contentPadding = PaddingValues(bottom = 66.dp),
```

Comment:
Can you change this to 120? Just for the sake of appearence

Reply:
I agree, done